### PR TITLE
Updated column selector to use new responsive mixin

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<button
+  class="sky-btn sky-btn-default"
+  type="button"
+  [attr.aria-label]="'skyux_grid_columns_toolbar_button' | skyLibResources"
+  [title]="'skyux_grid_columns_button' | skyLibResources"
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html:5]: Shouldn't title and aria-label be the same?

File: src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+>
+  <sky-icon icon="columns"></sky-icon>
+  <span class="sky-column-selector-action-btn-text">
+    {{ 'skyux_grid_columns_toolbar_button' | skyLibResources }}
+  </span>
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html:14]: ☝️ There's a few attributes in here that need to be on new lines.

**Review by Blackbaud-AlexKingman — CHANGES_REQUESTED** [review]

File: src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html:5]: Good call. I believe that `skyux_grid_columns_button` is correct as it is `Choose columns` while the other label is simply `Column`. I have updated it to be the same.

File: src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html:14]: Done

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**